src/models/analytics/analytics-tables.ts

```diff
@@ -532,6 +561,8 @@ const analyticsTablesL10n: Record<TableKeys, AnalyticsTable> = {
         createdAt: 'Aangemaakt op',
         updatedAt: 'Bijgewerkt op',
         deletedAt: 'Verwijderd op',
+        pinGroupSort: 'NL Order in pin group',
```

Comment:
Please help me with a translation (maybe with another version in English too if you have a nicer idea)